src/components/common/ColonyHome/ColonyExtensions/ColonyExtensions.tsx

```diff
@@ -2,34 +2,33 @@ import React from 'react';
 import { defineMessages, FormattedMessage } from 'react-intl';
 
 import Heading from '~core/Heading';
-import { Colony, useColonyExtensionsQuery } from '~data/index';
+import { useColonyExtensionsQuery } from '~data/index';
 import { MiniSpinnerLoader } from '~core/Preloaders';
 import extensionData from '~data/staticData/extensionData';
-
-import styles from './ColonyExtensions.css';
 import NavLink from '~core/NavLink';
 import ExtensionStatus from '~dashboard/Extensions/ExtensionStatus';
+import { useColonyContext } from '~hooks';
+
+import styles from './ColonyExtensions.css';
+
+const displayName = 'dashboard.ColonyHome.ColonyExtensions';
 
 const MSG = defineMessages({
   title: {
-    id: 'dashboard.ColonyHome.ColonyExtensions.title',
+    id: `${displayName}.title`,
     defaultMessage: 'Enabled extensions',
   },
   loadingData: {
-    id: 'dashboard.ColonyHome.ColonyMembers.loadingData',
+    id: `${displayName}.loadingData`,
     defaultMessage: 'Loading extensions data ...',
   },
 });
 
-interface Props {
-  colony: Colony;
-}
-
-const displayName = 'dashboard.ColonyHome.ColonyExtensions';
+const ColonyExtensions = () => {
+  const { colony } = useColonyContext();
 
-const ColonyExtensions = ({ colony: { colonyName, colonyAddress } }: Props) => {
   const { data, loading } = useColonyExtensionsQuery({
-    variables: { address: colonyAddress },
+    variables: { address: colony?.colonyAddress },
   });
```

Comment:
This won't exist anymore, as extensions will be fetched alongside the main colony object.

But you can leave it here for now. We'll refactor it when we get to it.